test/cli/remote-table-upload/table-create-columns-flag/args

```diff
@@ -0,0 +1 @@
+remote table upload --storage-api-host %%TEST_KBC_STORAGE_API_HOST%% --storage-api-token %%TEST_KBC_STORAGE_API_TOKEN%% --primary-key Id --file-name data.csv --file-tags test in.c-my-bucket.data data.csv --columns Id,Name,Region,Status,First_Order
```

Comment:
This usage is ok. I would not restrict it. You enforce user's to adjust the `.csv` file to remove `header` section.

Reply:
[7dbf765](https://github.com/keboola/keboola-as-code/pull/1841/commits/7dbf765a29ed8b01a93b947b5744f7fdaa44536f)